docs/en/observability/profiling-upgrade.asciidoc

```diff
@@ -5,6 +5,9 @@
 <titleabbrev>Upgrade</titleabbrev>
 ++++
 
+This section is specific to upgrading Universal Profiling when running it in {ecloud} or ECE.

```

Comment:
```suggestion
This section is specific to upgrading Universal Profiling on {ecloud} or ECE.

```

Reply:
Question here: is it ok to use both `{ecloud}` and `ECE`, or is there a placeholder for ECE? Or even, ECE docs are generated by using a different placeholder for `{ecloud}` so we should only have that in this sentence?

Reply:
There is an {ece} attribute as well. The list of attributes is actually available [here](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc), and it's ok to use it here.

Reply:
Thanks, updating it